graveyard.json

```diff
@@ -171,7 +171,7 @@
     "dateClose": "2006-05-08",
     "dateOpen": "2003-11-10",
     "description": "Google Deskbar was a small inset window on the Windows toolbar and allowed users to perform searches without leaving the desktop.",
-    "link": "https://www.computerweekly.com/news/2240053391/Google-launches-Deskbar-software",
+    "link": "https://gcemetery.co/google-deskbar/",
```

Comment:
Sorry, we're not going to be linking to a competitor as a source. Especially when that competitor ripped off Killed by Google. :)